pkgs/applications/networking/protonvpn-gui/default.nix

```diff
@@ -37,12 +38,16 @@ buildPythonApplication rec {
     wrapGAppsHook3
   ];
 
-  buildInputs = lib.optionals withIndicator [
-    # Adds AppIndicator3 namespace
-    libappindicator-gtk3
-    # Adds AyatanaAppIndicator3 namespace
-    libayatana-appindicator
-  ];
+  buildInputs =
+    [
+      libnotify # gir
```

Comment:
Maybe I'm missing something, but what do you mean by the gir comment?

Reply:
Changed to the `gir typelib is used`